include/bluetooth/services/bas_c.h

```diff
@@ -192,6 +213,39 @@ int bt_gatt_bas_c_read(struct bt_gatt_bas_c *bas_c, bt_gatt_bas_c_read_cb func);
  */
 int bt_gatt_bas_c_get(struct bt_gatt_bas_c *bas_c);
 
+/**
+ * @brief Check if notification is supported by service.
+ *
+ * @param bas_c BAS Client object.
+ *
+ * @return true If notifications are supported.
+ *              Otherwise false is returned.
+ */
+static inline bool bt_gatt_bas_c_notify_supported(struct bt_gatt_bas_c *bas_c)
+{
+	return bas_c->notify;
+}
+
+/**
+ * @brief Read periodically the battery level value from the device with
+ *        specific time interval.
+ *
+ * This function sends a read request to the connected device periodically.
+ * Can be used only when BAS do not support notification. Time interval
+ * K_NO_WAIT or K_FOREVER causes reading charasteristic value as soon as possible.
+ * Interval can be changed during reading If reading procces is in progress then
```

Comment:
This sentence is a bit weird, I would suggest:
"Periodic read interval can be changed while periodic read is active, the next read period will be started with the new interval."

In any case spelling of "procces". Unexpected upper case on "If", and I think "reading" should be "read"

Reply:
Ok changed